ssd_topic_name : /person_follower/objects_rect
cloud_topic_name : /camera/depth/points
laser_topic_name : /scan
target_frame : base_footprint